composer.json

```
@@ -1,27 +1,33 @@
 {
-	"name": "gathercontent-importer",
-	"description": "Imports items from GatherContent to your wordpress site",
-	"version": "3.0.0",
-	"type": "wordpress-plugin",
-	"keywords": [],
-	"homepage": "http://www.gathercontent.com",
-	"license": "GPL-2.0+",
-	"authors": [
-		{
-			"name": "GatherContent",
-			"email": "support@gathercontent.com",
-		"homepage": "http://www.gathercontent.com",
-			"role": "Developer"
-		}
-	],
-    "minimum-stability": "dev",
-	"require": {
-		"php": ">=5.6"
-	},
-    "require-dev": {
-	    "antecedent/patchwork": "1.2.*",
-        "phpunit/phpunit"     : "*@stable",
-	    "brianium/paratest"   : "dev-master",
-        "10up/wp_mock"        : "dev-master"
+  "name": "gathercontent/wp-importer",
+  "description": "Imports items from GatherContent to your wordpress site",
+  "version": "3.2.0",
```

```suggestion
  "version": "3.1.0",
```

Any reason to bump this two versions, and not just one?